spec/cog_cmd/ecs/container/update_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe 'the container-update command' do
+  let(:command_name) { 'container-update' }
+
+  let(:client) do
+    client = Object.new
+    allow(Aws::S3::Client).to receive(:new).and_return(client)
```

Comment:
This is probably better suited for a `before` block. The result would look something like this.

```
let(:command_name) { 'container-update' }
let(:client) { Object.new }

before do
  allow(Aws::S3::Client).to receive(:new).and_return(client)
end
```